compiler/AST/astutil.cpp

```diff
@@ -887,3 +887,25 @@ Symbol* getSvecSymbol(CallExpr* call) {
     return NULL;
   }
 }
+
+/*
+* Collect all of the function symbols that belong to function calls
+* and nested function calls that occur from baseAST. In other words
```

Comment:
You might want to rework this comment for clarity:
I think you mean "a list of all functions in the call graph at and below this function".  "Nested function" has a special meaning in Chapel: a function that is declared within the body of another function.